engine/access/state_stream/backend/handler.go

```diff
@@ -9,28 +9,31 @@ import (
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 
+	"github.com/onflow/flow-go/engine/access/state_stream"
 	"github.com/onflow/flow-go/engine/common/rpc"
 	"github.com/onflow/flow-go/engine/common/rpc/convert"
 	"github.com/onflow/flow-go/model/flow"
 )
 
 type Handler struct {
-	api   API
+	api   state_stream.API
 	chain flow.Chain
 
-	eventFilterConfig EventFilterConfig
+	eventFilterConfig state_stream.EventFilterConfig
 
-	maxStreams  int32
-	streamCount atomic.Int32
+	maxStreams        int32
+	streamCount       atomic.Int32
+	heartbeatInterval uint64
```

Comment:
let's make the name more explicit about how it's used
```suggestion
	defaultHeartbeatInterval uint64
```